### DISCUSSION TITLE
pdf template:  div  float:left|right not working

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16989] [General] [answered]
pdf template:  div  float:left|right not working,  I'm sure the code is right,  but it could not work, is there a replacement solution?

**Comment by rskrzypczak** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForcePDF#supported-css-rules

**Reply by newminder** [reply]
okay, I use display: inline-block instead.  thank you very much. 
what if I want text-align: right in the div  how to make it?
